query-engine/core/src/interpreter/formatters.rs

```diff
@@ -9,7 +11,7 @@ pub fn format_expression(expr: &Expression, indent: usize) -> String {
             .collect::<Vec<String>>()
             .join("\n"),
 
-        Expression::Query { query } => match query {
+        Expression::Query { query } => match query.deref() {
```

Comment:
isn't that equivalent to `*query`?

Reply:
Just tried poking at this again, and apparently the right incantation is `match &**query`. Not sure I love it, but it saves us an import so I guess it's fine.

Really wish we had `box` in patterns so we could match on:

```rust
match query {
    box Query::Read(rq) => add_indent(indent, format!("{}", rq)),
    box Query::Write(wq) => add_indent(indent, format!("{}", wq)),
}
```
But that's still a bit out. Changed it to `&**` for this PR!